scourgify/address_constants.py

```diff
@@ -593,7 +601,202 @@
     'UPPER': 'UPPER',
     '#': '#'
 }
-
+LONGHAND_STREET_TYPES = {
```

Comment:
It is not an inversion of `STREET_TYPE_ABBREVIATIONS`.  If you look through `STREET_TYPE_ABBREVIATIONS` you will see that most abbreviations have several possible, common misspellings or irregular abbreviations that translate to one correct abbreviation.  But each abbreviation only translates to one single correct long form.
For example, `STREET_TYPE_ABBREVIATIONS` has the following mapping for `ST`.
```
'ST': 'ST',
'STR': 'ST',
'STREET': 'ST',
'STRT': 'ST',
```
But for long form, `ST` should only translate to `STREET`.  

So `LONGHAND_STREET_TYPES ` creates the mapping that allows the code to pick the one, and only one, correct long form street type.  And I can't simply use `LONGHAND_STREET_TYPES` as a bi-directional lookup, because that does not handle the additional alternate forms that may be found in the address being submitted that needs to be normalized either to the standard abbreviation or the standard long form, depending on the desired action.

Reply:
The same pattern I described above for directionals would work here, a single object with a "valid" abbreviation, a "valid" long hand name, and then a list of other variations.

Reply:
This faces the same issue as above for directionals.  As I mentioned there, this package does not directly parse an address string.  If I did then `123 Circle Crt` would come out `123 CIR CT` instead of `123 CIRCLE CT` or `123 CIRCLE COURT`.
Since the address has to be parsed before any standardization happens, then pulling the standardized result from the existing constants is a simple get action (`STREET_TYPE_ABBREVIATIONS['CRT']`) instead of a search, which is what the `next(....)` code above is doing.
Then if the long_hand version is requested for the output, because the default behavior is to output the abbreviated version, meaning I already have the standardized abbreviation, it is a simple get again `LONGHAND_STREET_TYPES['CT']` to find the long hand output value.